app/src/main/java/com/portfolio/tasky/data/RequestInterceptorApiKey.kt

```diff
@@ -1,14 +1,13 @@
-package com.portfolio.tasky.networking
+package com.portfolio.tasky.data
 
 import com.portfolio.tasky.BuildConfig
 import okhttp3.Interceptor
 import okhttp3.Response
 import java.io.IOException
 import javax.inject.Inject
 
+class RequestInterceptorApiKey @Inject constructor() : Interceptor{
```

Comment:
This is just my personal preference, but I like to have the name of the interface at the end

Ex:
RequestApiKeyInterceptor

Or better yet
ApiKeyInterceptor

Reply:
I agree 👍